graphql_api/schema/thing.py

```diff
@@ -44,6 +49,46 @@ def resolve_files(root, info, **args):
             db_metrics.put_duration(__name__, 'resolve_files[legacy]' , dt.utcnow()-t0)
 
         return res
+    
+    def resolve_parents(root, info, **args):
```

Comment:
@benjamineac my concern about having this here is that we're saying that the parent<->child relationship is represented as a TaskTaskRelationConnection. 

IMO Thing is super primitive, with no built in relationships except the files. Whereas parents & children are attributes of a Task (which should be a subclass of Thing).